en/user_guide/media.rst

```diff
@@ -0,0 +1,59 @@
+Media Management
+================
+
+Open Orchestra allows to manage media in the media library panel.
+
+Folder
+------
+
+All files are organised inside folders. It's possible to browse the folder tree to see files by clicking on the left panel.
+
+Create
+~~~~~~
+
+Creating a folder is done by clicking on the "new folder" button in the left panel.
+
+The creation form needs:
+
+* name: name of the folder
+* list of websites: using the folder content
+
+Remove
+~~~~~~
+
+To remove a folder, select the folder and click on the trash button on the right panel.
+
+Manage files
+------------
+
+The add button in the folder's content allows to upload files.
+
+A trash icon appears on media preview hovering to deleting media.
+
+Click on media preview open a page with two panel: crop files and add meta information.
+
+Crop
+~~~~
+
+Image crop offers some options :
```

Comment:
Those formats are preset from an "out of the box" install. I think you should mentionned that they can be personalized on a custom install. Explaining how is not required here.

This will prevent users using a custom install to not understand why what you're telling is not what they're seeing.